.github/workflows.src/tests.inc.yml

```
@@ -315,10 +315,12 @@
     # Restore the artifacts and environment variables
 
     - name: Download shared artifacts
-      uses: actions/download-artifact@v3
+      uses: Wandalen/wretry.action@1.3.0
```

I think it's `v1.3.0` - but let's use hash with a tag comment like:
```suggestion
      uses: Wandalen/wretry.action@a163f62ae554a8f3cbe27b23db15b60c0ae2e93c  # v1.3.0
```

---

Why do you prefer the hash over the tag?

---

Oh, because the tag can be changed and force pushed?

---

Yeah I think so? Given the action providers vary a lot, using hashes would be more stable